code/modules/projectiles/guns/energy/energy_gun.dm

```diff
@@ -208,3 +229,17 @@
 	ammo_type = list(/obj/item/ammo_casing/energy/disabler/scatter/ultima, /obj/item/ammo_casing/energy/laser/ultima)
 	var/obj/item/modular_computer/integratedNTOS
 	var/NTOS_type = /obj/item/modular_computer/internal
+
+/obj/item/gun/energy/e_gun/e11
+	name = "E-11 hybrid energy rifle"
+	desc = "A hybrid energy gun fondly remembered as one of the worst weapons ever made. It hurts, but thats if you can even get hit with it."
```

Comment:
Slight grammar edit and some rewording for flow.

"A hybrid energy gun fondly remembered as one of the worst weapons ever made. It hurts, but that's only if it manages to hit its target."